CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Breaking. Added version checks for binary data. Requires to convert graph data or add v1 at the top of meta files.
 
+- Add debug mode to MultiWorkersConverter, using worker_count=0 will now disable multiprocessing and show error messages.
```

Comment:
How would anyone find about the debug mode? I don't see it in the code without reading PR description.